apps/web/components/eventtype/AIEventController.tsx

```diff
@@ -0,0 +1,287 @@
+            required
+            placeholder={t("phone_number")}
+            id="numberToCall"
+            value={numberToCall}
+            onChange={(val) => {
+              setNumberToCall(val);
+            }}
+          />
+
+          <Divider />
+        </>
+
+        <Label>{t("guest_name")}</Label>
+        <TextField
+          type="text"
+          hint="Variable: {name}"
+          label={t("guest_name")}
+          placeholder="Jane Doe"
+          value={guestName}
+          onChange={(e) => {
+            setGuestName(e.target.value);
+          }}
+        />
+
+        <Divider />
+        <TextAreaField
+          rows={3}
+          required
+          placeholder="General Prompt"
+          label="General Prompt"
+          {...formMethods.register("aiPhoneCallConfig.generalPrompt")}
+          onChange={(e) => {
+            formMethods.setValue("aiPhoneCallConfig.generalPrompt", e.target.value, { shouldDirty: true });
+          }}
+        />
+
+        <TextAreaField
+          rows={3}
+          placeholder="Begin Message"
+          label="Begin message"
+          {...formMethods.register("aiPhoneCallConfig.beginMessage")}
+          onChange={(e) => {
+            formMethods.setValue("aiPhoneCallConfig.beginMessage", e.target.value, { shouldDirty: true });
+          }}
+        />
+
+        {/*  Peer's comments here */}
+        {/* // TODO: v2 will allow users to create phone numbers and agents within Cal.com.
+            // we do this later, i just wanted to make the layout first
+            */}
+        {/* {v2 && (
+              <>
+                <Label>Guest Email</Label>
+                <TextField
+                  hint="Variable: {email}"
+                  label="Email of Guest"
+                  required
+                  placeholder="jane@example.com"
+                />
+              </>
+            )} */}
+
+        {/* {v2 && (
+              <>
+                <TextAreaField
+                  rows={3}
+                  value="Make it very casual and breath and say umm and things that sound very natural. Communicate concisely and conversationally. Short sentences ideally less than 20 words and have enough breaks in between sentences. Your approach should be empathetic and understanding, balancing compassion with maintaining a professional stance on what is best for the employee. Be convincing. Really try hard to get a meeting. Be decisive but also polite.  It is ok if you don't get a meeting on your calendar. If someone asks if you are a robot, you say yes. Deel.com is using an automated voice agent to help, but all of our scheduled meetings are with real humans, pinky promise."
+                  name="Behaviour"
+                />
+              </>
+            )} */}
+
+        <Button
+          disabled={createCallMutation.isPending}
+          loading={createCallMutation.isPending}
+          onClick={handleSubmit}>
+          Make a Call
+        </Button>
```

Comment:
NIT: I think this button should be renamed. I see "Make a call" I expect a call to happen on the frontend but I see it's just creating the LLM. This could be confusing.

Reply:
It is creating LLM and then making the call to "numberToCall"